Mailer/app/api/Validation/schemas/logs-count.schema.json

```diff
@@ -1,11 +1,51 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "description": "Logs count resource",
+  "description": "Get logs count",
   "type": "object",
+  "required": [
+    "filter"
+  ],
+  "dependencies": {
+    "page": ["limit"]
+  },
+  "additionalProperties": false,
   "properties": {
     "filter": {
+      "oneOf": [{
+        "type": "array",
+        "items": {
+          "enum": ["delivered_at", "clicked_at", "opened_at", "dropped_at", "spam_complained_at", "hard_bounced_at"]
+        }
+      },{
+        "type": "object",
+        "propertyNames": {
```

Comment:
The propertyNames validation is not working because the validator package supports only draft-04 of json schema. I can write validation for each column or we can find a newer package.

Reply:
In the end this was about `additionalProperties`, I see them now. Resolving.